tools/data_manipulation/xarray/xarray_mapplot.xml

```diff
@@ -0,0 +1,204 @@
+<tool id="xarray_mapplot" name="NetCDF xarray map plotting" version="0.18.2">
```

Comment:
if you add profile="20.05" I think the HOME will be set to your CWD automatically. Always use a recent profile if you can.